XamlControlsGallery/ControlPages/RichEditBoxPage.xaml.cs

```diff
@@ -34,14 +35,28 @@ public RichEditBoxPage()
         private void Menu_Opening(object sender, object e)
         {
             CommandBarFlyout myFlyout = sender as CommandBarFlyout;
-            if (myFlyout.Target == REBCustom)
+            if (myFlyout != null && myFlyout.Target == REBCustom)
             {
                 AppBarButton myButton = new AppBarButton
                 {
                     Command = new StandardUICommand(StandardUICommandKind.Share)
                 };
                 myFlyout.PrimaryCommands.Add(myButton);
             }
+            else
```

Comment:
What is the purpose of this code? It doesn't look to do much. Is the goal to expand on the RichEditBox sample to show how to integrate a CommandBarFlyout? Would this sample be better suited to the CommandBarFlyout page?

Reply:
It's added a share button to the commandbarflyout. If I remove `else`, it does have the share button

![image](https://user-images.githubusercontent.com/6290692/113340062-44d3e880-92e0-11eb-8436-4b186ab023a0.png)

Reply:
Right, one of the XCG samples shows how to add a custom button to the default text edit menu. That should indeed be an app-side fix.

But the original issue was that the text edit menus weren't showing at all. That problem I would have expected to be fixed in WinUI.

So maybe we need two unrelated PRs? XCG should be showing the default floaty at least in some samples.

Reply:
This sample should just work:
![image](https://user-images.githubusercontent.com/13142665/113341958-b7de5e80-92e2-11eb-9485-ded0fea0928d.png)

This sample may need an additional fix in XCG:
![image](https://user-images.githubusercontent.com/13142665/113342014-c9276b00-92e2-11eb-9117-8d94d1b4a587.png)

Reply:
I didn't see winui has problem. Maybe VS compiler issue?

I expect the app crashed when myFlyout is null, but it's not. Instead, XCG doesn't show the default floaty

You can verify it locally to change
```
if (myFlyout.Target == REBCustom)
```

to
```
if (myFlyout != null && myFlyout.Target == REBCustom)
```